src/helpers/helpers.core.js

```diff
@@ -30,8 +30,8 @@ export function inBoxRange(point, {x, y, width, height}, axis, borderWidth) {
 }
 
 export function getElementCenterPoint(element, useFinalPosition) {
-  const {x, y} = element.getProps(['x', 'y'], useFinalPosition);
-  return {x, y};
+  const {centerX, centerY} = element.getProps(['centerX', 'centerY'], useFinalPosition);
```

Comment:
> Would it be better for backward compatibility, if we used `x` and `y` for the center point? (and x1,y1 for the top/left)

make sense!

> On the other hand centerX and centerY are explicit. Could also go further explicit and use top, left, bottom, right for the corner coordinates.

Could make sense but in this way we don't have any backward compatibility, for any annotation type.  Do you want I do in this PR or with another one?

Reply:
@kurkle be also aware that top, left, bottom and right can not map very well the line annotation because bottom could be greater than top.

Reply:
> > Would it be better for backward compatibility, if we used `x` and `y` for the center point? (and x1,y1 for the top/left)
> 
> make sense!

@kurkle sorry but I think I misunderstood your comment. In my poor option it couldn't be a good idea to use `x,y` for center because it's not readable, I guess. The `getCenterPoint` method returns `{x: number, y: number}`.

Reply:
Not sure what you mean with "not readable"? The element properties can be read directly from the element. But if you mean its not obvious what x and y are, I agree.

I think its ok to go with centerX/Y, just wanted to bounce these thoughts.

Reply:
> Not sure what you mean with "not readable"? The element properties can be read directly from the element. But if you mean its not obvious what x and y are, I agree.

I meant "user friendly" or obvious (as you wrote). I didn't the user can not access to them ;)